Map/MapManager.gd

```diff
@@ -1,20 +1,21 @@
 extends Node2D
 ## Class to manage backend for rooms (generation and such)
  
-static func create_map() -> MapBase: ## Generates and Populates a map with rooms that have random room types. More in depth algorithms will be added in the future
+#layout changes the width of the map's floors
+static func create_map(_layout: Array[int]) -> MapBase: ## Generates and Populates a map with rooms that have random room types. More in depth algorithms will be added in the future
```

Comment:
I think `floors_width` was clear, why change to `_layout` which can describe more things ? to not be confused with the width of the map in itself ?
if that's the case, i would like to have a name that is close